flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/networking/NetworkFailureHandler.java

```diff
@@ -92,7 +92,14 @@ public void channelOpen(ChannelHandlerContext context, ChannelStateEvent event)
 		final Channel sourceChannel = event.getChannel();
 		sourceChannel.setReadable(false);
 
-		if (blocked.get()) {
+		boolean isBlocked = blocked.get();
+		LOG.debug("Attempt to open proxy channel from [%s] to [%s:%s] in state [blocked = %s]",
```

Comment:
Is there a particular reason why you use `%s` instead of `{}` (the latter being more common)?